src/main/app/claims/responseModelConverter.ts

```diff
@@ -92,4 +95,9 @@ export class ResponseModelConverter {
     return party
   }
 
+  private static convertWhenDidYouPay (paymentDeclaration: DraftPaymentDeclaration): PaymentDeclaration {
+    if (paymentDeclaration !== undefined) {
```

Comment:
What will get returned by that method if `paymentDeclaration !== undefined` evaluates to false? Can you please explicitly define returned value?